RecoParticleFlow/PFProducer/python/pfLinker_cff.py

```diff
@@ -2,10 +2,10 @@
 
 import RecoParticleFlow.PFProducer.pfLinker_cfi
 import RecoParticleFlow.PFProducer.particleFlowTmpPtrs_cfi
-particleFlow = RecoParticleFlow.PFProducer.pfLinker_cfi.pfLinker.clone()
-particleFlow.PFCandidate = [cms.InputTag("particleFlowTmp")]
+particleFlowPrePuppi = RecoParticleFlow.PFProducer.pfLinker_cfi.pfLinker.clone()
+particleFlowPrePuppi.PFCandidate = [cms.InputTag("particleFlowTmp")]
 particleFlowPtrs = RecoParticleFlow.PFProducer.particleFlowTmpPtrs_cfi.particleFlowTmpPtrs.clone()
-particleFlowPtrs.src = "particleFlow"
```

Comment:
this should now be moved to RecoParticleFlow_cff.py, where the particleFlow module is defined